doc-content/enterprise-only/openshift/proc-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,77 @@
+[id="proc-dashbuilder-standalone_{context}"]
+= Installing Dashbuilder Standalone using Operator
+
+You can use Operator to install Dashbuilder Standalone independently from other services.
```

Comment:
Don't hate me. Is there an alternate way to install DS? If not, you can shorten to:

Use Operator to install Dashbuilder Standalone independently from other services.

Reply:
You're right, it should be "separately" and only "independent" comparing Dashbuilder Standalone to Dashbuilder component.